modules/microshift-accessing-cluster-remotely-non-admin.adoc

```diff
@@ -25,6 +23,17 @@ Use the following procedure to access the {product-title} cluster from a remote
 [user@microshift]$ sudo firewall-cmd --permanent --zone=public --add-port=6443/tcp && sudo firewall-cmd --reload
 ----
 
+.Verify {product-title} is running
+
+. As `user@microshift`, verify that {product-title} is running by running the following command:
++
+[source,terminal]
+----
+[user@workstation]$ oc get all -A
+----
+
+.Procedure
+
 . Optional: As `user@workstation`, to create a `~/.kube/` folder if your {op-system} machine does not have one, run the following command:
```

Comment:
This step isn't optional: What the `mkdir -p ~/.kube/` command does is it create the directory if it didn't exist before and does nothing if it did exist. Means, no matter what, after running the command the directory exists and we can proceeed. I'd simply describe this as:

"As `user@workstation`, ensure a `~/.kube/` folder exists on your `microshift` host by running the following command:"